main.py

```diff
@@ -33,6 +32,21 @@ def initialize_players(number_of_players, deck):
     return players
 
 
+def find_most_common_color(cards):
+    cards_colors = [card.color for card in cards if card.color != "change color"]
```

Comment:
There are two way to get all the cards_colors, 
Backlist -> get all the card except not card color
white list -> filter only on the card color. 

a white list is better because you explain explicitly what you looking for, and in case someone adds new card type (like stop card) you dont need to change your code.

Think about how you can design your code better to make it happens. 

(think how the comment about the color patch connect to that)